rfc/rfc005-distributed-network-using-grpc.md

```diff
@@ -179,16 +176,14 @@ on their node. However, when offences are repeated the node SHOULD apply "Three
 SHOULD deny further connections using the peer's certificate (identified by certificate issuer DN and serial number),
 until the ban is lifted by an operator.
 
-TODO: Limit number of connections from a certain certificate?
-
 ### 8.1. Denial of Service
 
 #### Threat: Memory Exhaustion due to Large Messages
 
-A (malicious) peer could exhaust the node's memory with (many) large messages.
+A (malicious) peer could exhaust the node's memory with (many) large network messages.
 
 Countermeasures:
-- Nodes SHOULD NOT accept incoming messages larger than 5mb (5242880 bytes).
+- Nodes MUST NOT accept incoming network messages larger than 5mb (5242880 bytes).
```

Comment:
That's still quite large, we may need to re-evaluate in the future.

Reply:
I think we can safely limit it to 512kb, given that the message that can be the largest is the transaction list of a block. We can always make the limit higher.